sdl-snowflake/src/main/scala/io/smartdatalake/smartdatalake.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2021 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.smartdatalake
```

Comment:
It's not clean if you define an object in module sdl-snowflake on the level of io.smartdatalake. We should move this to a package io.smartdatalake.utils.snowflake, or to the language implementation you are working on...

Reply:
This is a [package object](https://docs.scala-lang.org/tour/package-objects.html) that contains package-wide type aliases for the io.smartdatalake package in the sdl-snowflake module, as described in the link. I think that the type aliases should be available in the entire package "io.smartdatalake" in the sdl-snowflake module, not inly in the "io.smartdatalake.utils.snowflake" package.

Of course, the type aliases are not strictly necessary, but I found that it makes working in classes with mixed Spark/Snowpark namespaces a lot easier.

Can you maybe elaborate a bit more on why you think this approach is not clean?